lib/pokemon-site.js

```diff
@@ -24,11 +24,21 @@ class PokemonSite {
     this._app = app
 
     // Start listening
-    const server = http.createServer(app)
-    server.listen(config.listenPort, config.listenAddress, () => {
+    this._server = http.createServer(app)
+    this._server.listen(config.listenPort, config.listenAddress, () => {
       console.log('Listening on %s:%d ...', config.listenAddress, config.listenPort)
     })
-    this._server = server
+
+    // Keep track of open connections
+    this.socketId = 0
+    this.sockets = {}
+    this._server.on('connection', (socket) => {
+      let socketId = ++this.socketId
```

Comment:
@johartl the importance is consistency. It is a common practice (and a best practice) to start indexing at 0. So unless you have a very good reason to divert from convention, it is best that you just stay with the current.